### ISSUE TITLE
log tasks with Markdown format (Instead of plain text)

### ISSUE THREAD

**Issue by eslam-gomaa** [eslam-gomaa/Flexible-Network#36]
As a result of this when logs are sent via Slack or RocketChat they will be displayed in Markdown instead of plain text

- [x] print logs in Markdown (with `rich` library)
- [x] Make an option to choose between plain text and markdown

---

**Blocked by:**

- [x] #38

---
**Preferred to be done with**
- [ ] #28

**Comment by eslam-gomaa** [comment]
Done #43 

- Options:
  - `markdown`  (default)
  - `txt`


**Example Yaml**

> log_format

```yaml
Task:
  name: New Task
  log_format: markdown
  subTask:
    - name: Love forever
      vendor: huawei
      parallel: false
      authenticate:
        # host: 192.168.1.16
        # label: switches 
        group: works
        port: 1113
        username: orange
        password: cisco
        reconnect: false
      configBackup:
        comment: "Test backup"
        after_commands: false
        target: local
        skip:
          - 192.168.1.11
          - 192.168.1.12
      commands:
        - command: show ip int br
          tag: show1
          ask_for_confirmation: false
          exit_on_fail: false
          onlyOn:
            - 90.84.41.239            
        - command: |
            sh vlan br
            show version
            show ip int
          when:
            tag: show1
            operator: is
            exit_code: 0

```

---

**If using the library**

```
task = Terminal_Task(task_log_format='markdown'))
```
